modules/trusted-firmware-m/Kconfig

```diff
@@ -229,11 +229,17 @@ config TFM_IPC
 	  TF-M Profile option is supplied, do not allow manual setting of
 	  the IPC mode, as it is determined by the profile setting.
 
-config TFM_REGRESSION
-	bool "Regression tests"
+config TFM_REGRESSION_S
+	bool "Regression tests Secure"
 	help
 	  When enabled, this option signifies that the TF-M build includes
-	  the Secure and the Non-Secure regression tests.
+	  the Secure domain regression tests.
+
+config TFM_REGRESSION_NS
+	bool "Regression tests Non-Secure"
+	help
+	  When enabled, this option signifies that the TF-M build includes
+	  the Non-Secure domain regression tests.
```

Comment:
This help text (and maybe the promt also) should make it clear that the Zephyr app is completely replace with TF-M's NS app.

Reply:
Yes, I agree about this. We need to stress that @andvib

Reply:
Good point! I tried rephrasing both these and the README for this to be clearer